data/korath/korath missions.txt

```diff
@@ -257,3 +257,81 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Efret: Quarg Rescue"
+	boarding
+	repeat
+	invisible
+	deadline 1
+	source
+		government "Kor Efret"
+	on offer
+		"old quarg rep" = "reputation: Quarg"
+		# bring it low enough that capping this will result in war
+		"reputation: Quarg" <?= 1
+		conversation
+			branch "unfriendly quarg" # if you're already at war with them
+				"reputation: Quarg" < 0
+			`The Quarg have brought reinforcements, probably in reaction to a distress call by the <ship>. They hail you, with barely contained anger in their voice. "Remove thyself from hence, and we may yet forgive thy aggression. The Kor Efret art under our protection! Doth not e'en think about capturing yond vessel."`
+			choice
+				`  "I had not realized they were under your protection. I will leave at once."`
+				`  (Ignore the warning.)`
+					goto ignore
+				`  "I think you should leave. This is my prize and I'll take what I please."`
+					goto refuse
+			`  "Leaveth then anon, ere we changeth our mind."`
```

Comment:
Maybe this dialog should be changed a bit since they shoot you on the way out?

Reply:
I did, down on line 292?  Did I not label it right?

Reply:
Should probably have the quarg ships that appear not be entering though for this one....

Reply:
Well they say to leave before they change their minds, but they are shooting you all the same

Reply:
Shouldn't that line be skipped by the `goto refuse` ?